rpcserver.go

```diff
@@ -2322,7 +2322,7 @@ func unmarshalLeafKey(key *unirpc.AssetKey) (universe.BaseKey, error) {
 
 	switch {
 	case key.GetScriptKeyBytes() != nil:
-		pubKey, err := schnorr.ParsePubKey(
+		pubKey, err := btcec.ParsePubKey(
```

Comment:
Doesn't match this: https://github.com/lightninglabs/taproot-assets/blob/dc5782b0bd0eb8b97b68fc9c50274e1fc8fa8db8/rpcserver.go#L2206-L2217